src/Badges.t.sol

```diff
@@ -1296,6 +1341,46 @@ contract BadgesTest is Test {
     assertEq(badgesWrappedProxyV1.balanceOf(recipient3), 0);
   }
 
+  function testGiveRequestedBadgeToManyWithBadSignature() public {
+    address active = raftHolderAddress;
+    address recipient1 = claimantAddress;
+    uint256 recipient1PrivateKey = claimantPrivateKey;
+    address recipient2 = passiveAddress;
+    uint256 recipient2PrivateKey = passivePrivateKey;
+    address recipient3 = vm.addr(randomPrivateKey);
+    createRaftAndRegisterSpec();
+    bytes memory recipient1Signature = getSignatureForRequestedBadge(
+      recipient1,
+      recipient1PrivateKey
+    );
+    bytes memory recipient2Signature = getSignatureForRequestedBadge(
+      recipient2,
+      recipient2PrivateKey
+    );
+    bytes memory recipient3Signature = bytes("bad signature");
+    address[] memory recipientsAddresses = new address[](3);
```

Comment:
Sometimes this is rather hard to follow what the test is actually doing as it reads quite verbose. Is there a way we can achieve more brevity here via refactoring? Not criticial for the context of this PR